### DISCUSSION TITLE
Issue in comment

### DISCUSSION THREAD

**Discussion by coindroid42** [rozpuszczalny/vscode-redmine#58] [Ideas]
When i write comment i use links with hashtag like 

`// #57`

1. It would be better if _vscode-redmine_ load title of the task and put in line

`// #57 Issue in comment`

2. It would be ideal if there was a text search interface and added a comment as link in the place where the cursor is located
3. Above the comment, we need to show the actions that we can take with the task

<sup><sub>Change status | Add time | Open in browser | Quick update</sub></sup> 
`// #57 Issue in comment`

**Comment by rozpuszczalny** [comment]
Hey, thanks for the suggestion!

It's an elegant idea! Before I can create a feature request issue, I'd like to ask you a few more questions:

1. Can you give me an example of another extension that adds more text to something in the code? I don't think adding plain text to the code is a good idea, and I need some more inspiration 😃 
2. What should happen, if there are multiple issue ids in a single line, e.g. `// #123, #456`? Should actions eyebrow be presented for the first issue? Or in that case, eyebrow should not be presented at all? I'm also thinking about a non-ideal solution, that would show a quick pick with issues defined in the line. And how to put additional text in there? If you have an idea of that, don't hesitate to share it here! 😄

**Reply by coindroid42** [reply]
Thankyou, 😊

1. Perhaps this will not write direct text to the file, but only modify the view:
- [naumovs.color-highlight](https://marketplace.visualstudio.com/items?itemName=naumovs.color-highlight)
- [liamhammett.inline-parameters](https://marketplace.visualstudio.com/items?itemName=liamhammett.inline-parameters)

🤔 Although adding text to a comment directly on a specific line, using a keyboard shortcut. it would be good for those who do not have the module installed. 

2. For one line, only one issue. like as this extension [Isotechnics.commentlinks](https://marketplace.visualstudio.com/items?itemName=Isotechnics.commentlinks) + _In this extension presents - actions above line_
// #123 issue title here
// #190 one more issue
